src/one_dragon/base/config/yaml_operator.py

```
@@ -6,8 +7,82 @@
+
+
+storeable_cache_config_prefixes = [
+    "assets/",
+    "config/",
+]
+
 cached_yaml_data: dict[str, tuple[float, dict]] = {}
 
+def flush_cache_to_file():
+    """
+    将缓存中符合前缀要求的配置存储到文件中，供下次启动时加载
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)